documentation/latest/dev/guide.md

```diff
@@ -100,27 +118,46 @@ HTTP response body: <html>
 > kubectl get ingress
 NAME              CLASS    HOSTS   ADDRESS   PORTS   AGE
 parodos-ingress   <none>   *                 80      9s
-
 ```
+
 ## Details
-_Parodos_ treats integration tests as unit tests and executes them independently using the Maven _Failsafe_ plugin instead of the classic _Surefire_ plugin.
 
-The Maven _Surefire_ and _Failsafe_ plugins are two widely used plugins for executing tests during a Maven build. These two plugins differ in the build lifecycle phase in which they operate. While the _Surefire_ plugin runs tests during the test phase, the _Failsafe_ plugin executes tests during the integration-test phase.
+_Parodos_ treats integration tests as unit tests and executes them independently
+using the Maven _Failsafe_ plugin instead of the classic _Surefire_ plugin.
 
-The _Failsafe_ plugin is designed to execute integration tests that require more elaborate setup, such as tests that rely on a running application server or database connection. Integration tests run in the same JVM instance as the application, and the plugin includes a default naming convention for identifying integration test classes. _Failsafe_ generates reports that provide insight into the overall health of the application, as well as any integration issues that may exist.
+The Maven _Surefire_ and _Failsafe_ plugins are two widely used plugins for
+executing tests during a Maven build. These two plugins differ in the build
+lifecycle phase in which they operate. While the _Surefire_ plugin runs tests
+during the test phase, the _Failsafe_ plugin executes tests during the
+integration-test phase.
 
-To facilitate running integration tests during the Maven build process, _Parodos_ defines a Maven profile named `integration-test`. The command to execute the integration tests is:
+The _Failsafe_ plugin is designed to execute integration tests that require more
+elaborate setup, such as tests that rely on a running application server or
+database connection. Integration tests run in the same JVM instance as the
+application, and the plugin includes a default naming convention for identifying
+integration test classes. _Failsafe_ generates reports that provide insight into
+the overall health of the application, as well as any integration issues that
+may exist.
+
+To facilitate running integration tests during the Maven build process,
+_Parodos_ defines a Maven profile named `integration-test`. The command to
+execute the integration tests is:
 
 ```bash
 mvn verify -pl integration-tests -P integration-test
 ```
-You can also specify the server IP and server port by setting the environment variables `SERVERIP` and `SERVERPORT`. By default, the values are `localhost` and `8080`, respectively. For instance, you can run the following command to specify custom values for the server IP and port:
+
+You can also specify the server IP and server port by setting the environment
+variables `SERVERIP` and `SERVERPORT`. By default, the values are `localhost`
```

Comment:
should it be `SERVER_IP` and `SERVER_PORT` or `SERVERIP`  and `SERVER_PORT` ?

Reply:
`SERVER_IP` and `SERVER_PORT`. I'll fix that, thanks